### ISSUE TITLE
Jade block comment not rendering correctly

### ISSUE THREAD

**Issue by tuckermcclure** [sintaxi/harp#434]
It appears the block comments in Jade don't work when certain characters show up.

For instance, the following .jade file renders correctly when using Jade itself, but not does render correctly using Harp:
```
html
  body
    p This is a block comment test.
    //
      Comment line 1
      t = 1:1000; % Comment line 2
      Comment line 3
    p Did that work?
```
The error I receive is:
```
- Unexpected token punc «:», expected punc «)»
```

**Comment by silentrob** [comment]
Perhaps we need to bump Jade in Harp? Are you comparing the same versions of Jade?

**Comment by silentrob** [comment]
Terraform Jade is 0.35.0
Jade Current is 1.9.2

**Comment by tuckermcclure** [comment]
I was comparing against the current version. A jade bump would be appreciated.

**Comment by kennethormandy** [comment]
Thanks for opening an issue! We have an upgrade for this queued up: https://github.com/sintaxi/terraform/pull/83

In the meantime, if it’s really critical for you, I believe you can use [npm shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap) to manually bump Jade.

**Comment by tuckermcclure** [comment]
Thanks, @kennethormandy.

- Tucker

**Comment by chharvey** [comment]
I know this issue is closed but I wanted to bring up another related problem without opening another issue.

The [block code](http://jade-lang.com/reference/code/) syntax, [released in Jade v1.11.0 (June 2015)](http://jade-lang.com/history/) is also throwing an error.

    - var x = { type: 'paragraph' , content: 'hello' }
    p= x.content
    //- shows <p>hello</p>

    -
      var x = {
        type: 'paragraph',
        content: 'hello'
      }
    p= x.content
    //- fails
